shared/cfg/guest-os/Linux/RHEL/8.cfg

```diff
@@ -0,0 +1,44 @@
+- 8:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    os_variant = rhel8
+    unattended_install.url:
+        #Running from http server, RHEL requires more memory
+        mem = 2048
+    nic_hotplug:
+        modprobe_module =
+    block_hotplug:
+        modprobe_module =
+    no unattended_install..floppy_ks
+    unattended_install, svirt_install:
+        kernel = images/${os_variant}-${vm_arch_name}/vmlinuz
+        initrd = images/${os_variant}-${vm_arch_name}/initrd.img
+        unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install:
```

Comment:
Incorrect indentation at here, looks like requiring an one-level left shifting for the rest part.

Reply:
@luckyh corrected thanks